dev/release/02-source.sh

```diff
@@ -123,6 +126,18 @@ if [ ${SOURCE_UPLOAD} -gt 0 ]; then
   echo ""
 fi
 
+# Create Pull Request and Crossbow comment to run Verification tasks
+if [ ${SOURCE_PR} -gt 0 ]; then
+  archery crossbow verify-release-candidate \
+  --base-branch=${maint_branch} \
+  --arrow-remote=https://github.com/apache/arrow \
+  --head-branch=${release_candidate_branch} \
+  --pr-title="WIP: [Release] Verify ${release_candidate_branch}" \
+  --pr-body="PR to verify Release Candidate" \
+  --version=${version} --rc=${rc} \
+  --arrow-token=${ARROW_GITHUB_TOKEN} --create-pr
```

Comment:
Could you indent continuous lines and sort alphabetically?

```suggestion
    --arrow-remote=https://github.com/apache/arrow \
    --arrow-token=${ARROW_GITHUB_TOKEN} \
    --base-branch=${maint_branch} \
    --create-pr \
    --head-branch=${release_candidate_branch} \
    --pr-body="PR to verify Release Candidate" \
    --pr-title="WIP: [Release] Verify ${release_candidate_branch}" \
    --rc=${rc} \
    --version=${version}
```

Reply:
Done